ci/jobs/scale-down.yml

```diff
@@ -0,0 +1,60 @@
+jobs:
+  - name: scale-down-development
+    max_in_flight: 1
+    plan:
+      - get: dataworks-aws-tarball-ingester
+        trigger: false
+      - .: (( inject meta.plan.terraform-bootstrap ))
+      - .: (( inject meta.plan.terraform-output ))
+      - .: (( inject meta.plan.scale-down ))
+        config:
+          params:
+            AWS_ROLE_ARN: arn:aws:iam::((aws_account.development)):role/ci
+
+  - name: scale-down-integration
+    max_in_flight: 1
+    plan:
+      - get: dataworks-aws-tarball-ingester
+        trigger: false
+      - .: (( inject meta.plan.terraform-bootstrap ))
+      - .: (( inject meta.plan.terraform-output ))
```

Comment:
Your scale down jobs are missing the `TF_WORKSPACE` param that the scale up jobs correctly have.